src/Microsoft.Dnx.ApplicationHost/Program.cs

```diff
@@ -242,23 +253,21 @@ private Task<int> ExecuteMain(DefaultHost host, string applicationName, string[]
             return EntryPointExecutor.Execute(assembly, args, host.ServiceProvider);
         }
 
-        private static void ThrowEntryPointNotfoundException(
+        private static void NotifyEntryPointNotfound(
             DefaultHost host,
             string applicationName,
             Exception innerException)
         {
             if (host.Project.Commands.Any())
             {
-                // Throw a nicer exception message if the command
-                // can't be found
-                throw new InvalidOperationException(
-                    string.Format("Unable to load application or execute command '{0}'. Available commands: {1}.",
+                Console.Error.WriteLine(string.Format("Unable to load application or execute command '{0}'. Available commands: {1}.",
```

Comment:
Then make a class in Sources that wraps up the exception and sets the property? Otherwise our `throw`s are going to get gross...

```csharp
var ex = new Exception("something");
ex.Data["suppressStackTrace"] = true
throw ex;
```